bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/live/TelldusLiveDeviceController.java

```diff
@@ -276,37 +285,66 @@ public void onRequest(TellstickDeviceEvent newDevices) {
     }
 
     <T> T callRestMethod(String uri, Class<T> response) throws TelldusLiveException {
+        LocalDateTime start = LocalDateTime.now();
         T resultObj = null;
         try {
-            for (int i = 0; i < MAX_RETRIES; i++) {
-                try {
-                    resultObj = innerCallRest(uri, response);
-                    break;
-                } catch (TimeoutException e) {
-                    logger.warn("TimeoutException error in get", e);
-                } catch (InterruptedException e) {
-                    logger.warn("InterruptedException error in get", e);
-                }
-            }
-        } catch (JAXBException e) {
-            logger.warn("Encoding error in get", e);
+            resultObj = innerCallRest(uri, response);
+        } catch (TimeoutException | InterruptedException | ExecutionException | JAXBException | XMLStreamException e) {
             logResponse(uri, e);
-            throw new TelldusLiveException(e);
-        } catch (XMLStreamException e) {
-            logger.warn("Communication error in get", e);
-            logResponse(uri, e);
-            throw new TelldusLiveException(e);
-        } catch (ExecutionException e) {
-            logger.warn("ExecutionException error in get", e);
+            monitorAdditionalRequest(start, LocalDateTime.now(), e);
             throw new TelldusLiveException(e);
         }
+        monitorAdditionalRequest(start, LocalDateTime.now(), null);
         return resultObj;
     }
 
+    private void monitorAdditionalRequest(LocalDateTime start, LocalDateTime end, @Nullable Throwable exception) {
```

Comment:
`Instant` would be less susceptible to changes in time-zone while measuring (before/after midnight).

Reply:
Good suggestion. Done.